src/helm-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Values.name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.name }}
+    spec:
+      automountServiceAccountToken: false
+      securityContext:
+        runAsGroup: 0
+        runAsUser: 2
+      containers:
+        - name: {{ .Values.name }}
```

Comment:
Here, that's a no brainer choice since we don't really care about the container name (the pod is monocontainer... So it's not hard to differentiate containers between themselves I guess lol)
So we put `.Chart.Name`, that's completely fine.